### PR TITLE
chore: fix theorem naming referencing `Units.oneSub`

### PR DESCRIPTION
**Pull request by j-loreaux**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by Ruben-VandeVelde** [comment]
It seems like there's no deprecated aliases added?

**Comment by j-loreaux** [comment]
oops, sorry, will do.

File: Mathlib/Analysis/NormedSpace/Units.lean [hunk 1]
```diff
@@ -112,19 +114,22 @@ theorem inverse_add (x : Rˣ) :
   rw [← x.val_add t ht, inverse_unit, Units.add, Units.copy_eq, mul_inv_rev, Units.val_mul,
     ← inverse_unit, Units.val_oneSub, sub_neg_eq_add]
 
-theorem inverse_one_sub_nth_order' (n : ℕ) {t : R} (ht : ‖t‖ < 1) :
+theorem inverse_oneSub_nth_order' (n : ℕ) {t : R} (ht : ‖t‖ < 1) :
```
Comment (fpvandoorn) [inline Mathlib/Analysis/NormedSpace/Units.lean:117]: Some of these lemmas don't actually contain `Units.oneSub`, so I'm not sure if camelCase is actually better. Are you sure you want to rename these?

In either case:
bors d+

Comment (j-loreaux) [reply Mathlib/Analysis/NormedSpace/Units.lean:117]: fair enough, I guess that's why they are named this way in the first place. I had originally thought it was just a consequence of porting issues. Closing this.

**Pull request closed without merging**